string yyyymmdd_hhmmss_string
byte[] unbounded_cam_data
